obspy/io/mseed/headers.py

```diff
@@ -523,15 +523,17 @@ class UDIFF(C.Union):
 clibmseed.ms_detect.restype = C.c_int
 
 clibmseed.msr_decode_steim2.argtypes = [
-    C.POINTER(C.c_int32), C.c_int,
+    C.c_void_p,
```

Comment:
I think this commit was done due to this report: http://tests.obspy.org/57634/ that was building this commit: 1f4d2a84e48f258648e976c335715150a5aa31b1

The reason.. dunno.. @mbyt?